packages/wallets/src/components/SentEmailContent/__test__/SentEmailContent.spec.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { PropsWithChildren } from 'react';
+import { APIProvider, AuthProvider } from '@deriv/api-v2';
+import { useDevice } from '@deriv-com/ui';
+import { fireEvent, render, screen } from '@testing-library/react';
+import useSendPasswordResetEmail from '../../../hooks/useSendPasswordResetEmail';
+import { ModalProvider } from '../../ModalProvider';
+import SentEmailContent from '../SentEmailContent';
+
+jest.mock('../../../hooks/useSendPasswordResetEmail', () => ({
+    __esModule: true,
+    default: jest.fn(),
+}));
+jest.mock('@deriv-com/ui', () => ({
+    ...jest.requireActual('@deriv-com/ui'),
+    useDevice: jest.fn(() => ({})),
+}));
+
+const wrapper = ({ children }: PropsWithChildren) => (
+    <APIProvider>
+        <AuthProvider>
+            <ModalProvider>{children}</ModalProvider>
+        </AuthProvider>
+    </APIProvider>
+);
+
+describe('SentEmailContent', () => {
+    beforeEach(() => {
+        (useDevice as jest.Mock).mockReturnValue({ isDesktop: true });
+        (useSendPasswordResetEmail as jest.Mock).mockReturnValue({
+            error: null,
+            sendEmail: jest.fn(),
+        });
+    });
+
+    afterAll(() => {
```

Comment:
nit: shouldn't this be afterEach since the mocks are added in a beforeEach block